src/config/constants.ts

```diff
@@ -19,9 +19,9 @@ export const MEMBER_AVATAR_MAIN_SIZE = 256;
 
 export const UrlSearch = {
   KeywordSearch: 's',
-  DisciplineTagSearch: 'discipline',
-  LevelTagSearch: 'level',
-  ResourceTypeTagSearch: 'resourceType',
+  DisciplineTagSearch: 'disciplines',
+  LevelTagSearch: 'levels',
+  ResourceTypeTagSearch: 'resourceTypes',
```

Comment:
Why were these changed ? We need to check that it still works with the search params. I remember that there was a difference between the search params in the URL and the parameters that are passed to the backend.